qa/hooks/commit-msg

```diff
@@ -4,7 +4,7 @@ COMMIT_MESSAGE="$(cat $MSG_FILE)"
 REGEX='^((ADPT)-[0-9]+):'
 if [[ $COMMIT_MESSAGE =~ $REGEX ]]; then
  TICKET=${BASH_REMATCH[1]}
- RESULT=$(curl -s https://taskana.atlassian.net/rest/api/3/issue/$TICKET)
+ RESULT=$(curl -s https://kadai.atlassian.net/rest/api/3/issue/$TICKET)
```

Comment:
is this URL correct?

Reply:
This URL is not used anymore, I will delete it